test/TestProjects/LRO-LowLevel/LRO-LowLevel.json

```diff
@@ -0,0 +1,140 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "LRO Service",
+    "description": "LRO Service",
+    "version": "1.0.0"
+  },
+  "host": "localhost:3000",
+  "schemes": [
+    "http"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "paths": {
+    "/test/resources": {
+      "post": {
+        "operationId": "DoSomethingSLRO",
+        "description": ".",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Resource"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "operationId": "DoSomethingLRO",
+        "description": ".",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Resource"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location",
+          "x-ms-long-running-reallyLong": true
```

Comment:
This option has not defined in https://github.com/stankovski/AutoRest/blob/master/Documentation/swagger-extensions.md, and never be used by a real service (no record in https://github.com/Azure/azure-rest-api-specs).
There also had a similar PR merged in Aug. 20 https://github.com/Azure/autorest.csharp/pull/1473 that you made.
I talked with swagger team and they have no idea about this feature, so would like give more information about this change? And to this kind of feature it definitely should be written down in our swagger spec.
Loop @lirenhe  and @akning-ms .

Reply:
Not sure whether @johanste @lmazuel @allenjzhang  whether know this change/feature for existing LRO swagger extension?